libclc/test/binding/ocl/bitselect.cl

```diff
@@ -0,0 +1,456 @@
+
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Autogenerated by gen-libclc-test.py
```

Comment:
Should we keep auto-generated tests under version control or just the script which generates them?

Reply:
The script takes a while to generate everything, it is only not meant to be constantly rerun as the builtins are not meant to change.